src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --blue-color: #00A7E1;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo img {
+  height: 40px;
+  position: absolute;
+  top: 10px;
+  left: 50px;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  list-style: none;
+  display: inline-block;
+  margin-right: 20px;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
```

Comment:
Avoid setting right margins on list items. Be consistent with your vertical margins and add only top or only bottom margin, but not both. In this case, consider using padding on the parent element instead.